### PR TITLE
[devtools-blog] DevTools architecture refresh: migrating DevTools to TypeScript

### PR DESCRIPTION
**Pull request by chybie**

### CODE REVIEW

**Review by mathiasbynens — APPROVED** [review]

**Comment by chybie** [comment]
@robdodson the review process seems not working yet. The approval still route to the usual one even with label `reviewers:devtools`

**Comment by chybie** [comment]
cc @TimvdLippe for preview

**Review by surma — APPROVED** [review]

File: site/en/blog/migrating-to-typescript/index.md
Comment (TimvdLippe) [inline site/en/blog/migrating-to-typescript/index.md:19]: Nit: this sentence is missing a closing parenthesis.

File: site/en/blog/migrating-to-typescript/index.md
Comment (TimvdLippe) [inline site/en/blog/migrating-to-typescript/index.md:79]: Unfortunately the backticks are not really visible, but that's more of a stylistic comment on the rendering on the website rather than the contents of this post I suppose.

File: site/en/blog/migrating-to-typescript/index.md
Comment (TimvdLippe) [inline site/en/blog/migrating-to-typescript/index.md:142]: These asterisks remained in the eventual output. Do we need a newline before it?

**Review by TimvdLippe — APPROVED** [review]
Thanks!

File: site/en/blog/migrating-to-typescript/index.md
Comment (chybie) [reply site/en/blog/migrating-to-typescript/index.md:79]: Yea, will talk to the team on the styling to make it more visible.

**Comment by robdodson** [comment]
> @robdodson the review process seems not working yet. The approval still route to the usual one even with label reviewers:devtools

Hey @chybie, the devtools team has the ability to approve if the `reviewers: devtools` label is applied, but [they are currently set to not receive emails](https://github.com/GoogleChrome/developer.chrome.com/blob/907b77f321c0e330674366dad0208ee4d6485b3f/.pullapprove.yml#L45-L47). This is to cut down on notification spam. So the idea would be that you would add the label and then manually assign whoever from the team you wanted to review. But we can easily change it if that workflow doesn't work for y'all.

Can you tell me which workflow you would prefer? We can auto assign to specific folks, or have it rotate through the members of [the devtools team](https://github.com/orgs/GoogleChrome/teams/dcc-devtools).

**Comment by chybie** [comment]
No email is fine. 

For this PR, I tried to get Mathias to approve (devtools reviewers) after adding the `reviewers: devtools` label. However I could not merge it until I got Surma to approve it.

**Pull request merged**
